libkbfs/data_types.go

```diff
@@ -572,48 +572,60 @@ type BlockChanges struct {
 // BlockChanges.  Currently does not check for equality at the
 // operation level.
 func (bc BlockChanges) Equals(other BlockChanges) bool {
-	if bc.Info != other.Info || len(bc.Ops) != len(other.Ops) ||
-		bc.sizeEstimate != other.sizeEstimate {
+	if bc.Info != other.Info || len(bc.Ops) != len(other.Ops) {
```

Comment:
this is because either sizeestimate can be 0, right? Perhaps check that, if both are non-zero, that they're equal?